src/theme.js

```diff
@@ -2,7 +2,13 @@ const theme = {
   color: {
     serenity: '#92A8D1',
     arctic: '#63888E',
-    majorBrown: '#5B5149'
+    majorBrown: '#5B5149',
+    white: '#FFFFFF'
+  },
+  media: {
+    phoneLarge: '@media (min-width: 375px)',
```

Comment:
As we only have one size for this, would a more general name like `mobile` be better here?